### DISCUSSION TITLE
Organization is not verifying

### DISCUSSION THREAD

**Discussion by shiva-sandupatla** [community/community#22441] [New to GitHub]
<p>I have added my domain to organization at the section of <strong>verified domains</strong> and added the <strong>txt record</strong> in my domain dns as per it’s instruction.</p>
<blockquote>
<p>I have purchased my domain at <strong>google domains</strong></p>
</blockquote>
<p>after 1 hour when I click on verify to verify my organization it says,</p>
<p><code>We couldn't find the TXT record. Note that DNS changes can take up to 72 hours.</code></p>
<p>then i waited 72 hours, after that 72 hours also it says same as above :point_up:</p>
<p>is this problem for me only, or else is there any other way to do so,</p>
<p>Hoping with a positive response,<br>
<strong>Regards, shiva</strong></p>

**Comment by lenoth** [comment]
<p>This is happening to me too. Did you find a fix for it or is it still not verifying on your end?</p>

**Comment by shiva-sandupatla** [comment, marked as answer]
<p>Yes, i found a fix <a class="mention" href="/lenoth">@lenoth</a></p>
<p>while adding the <code>TXT</code> record</p>
<p>the github gives url fromat as <code>_github-challenge-&lt;your-github-username&gt;.&lt;yourdomain.com&gt;</code> to add in the <code>TXT</code> record and value will be something</p>
<p>so while adding in your dns remove the part of <code>.&lt;yourdomain.com&gt;</code> and add it.</p>
<p>for example:</p>
<p>my github organization username is <strong>“test”</strong> and my domain to verify is <code>myexample.com</code></p>
<p>so the github provides txt record as <code>_github-challenge-test.myexample.com</code> as the value will be something.</p>
<table>
<tbody>
<tr>
<td>Type</td>
<td>Host</td>
<td>value</td>
</tr>
<tr>
<td>TXT</td>
<td>_github-challenge-test</td>
<td> some value gitven by github </td>
</tr>
</tbody>
</table>
<p>Hope you understood,</p>
<p>Regards, shiva</p>

**Reply by Jupiternium** [reply]
WHERE DO I FIND THIS
PLEASEEE I NEED HELP

**Reply by shiva-sandupatla** [reply]
Can you explain in detail what you need to find

**Reply by dvd-22** [reply]
THANK YOU! I was extremely confused

**Comment by lenoth** [comment]
<p>Support told me the same thing and it worked. Thank you! :slight_smile:</p>

**Comment by jtara** [comment]
<p>Yes, the instructions are bad, I knew to remove the actual domain name from the record. But still doesn’t work for me. Have never been able to get this to work, guess I am still missing something.</p>

**Comment by shiva-sandupatla** [comment]
<p>What domain provider you are using?</p>

**Comment by Accuweaty24** [comment]
> I have added my domain to organization at the section of **verified domains** and added the **txt record** in my domain dns as per it’s instruction.
> 
> > I have purchased my domain at **google domains**
> 
> after 1 hour when I click on verify to verify my organization it says,
> 
> `We couldn't find the TXT record. Note that DNS changes can take up to 72 hours.`
> 
> then i waited 72 hours, after that 72 hours also it says same as above ☝️
> 
> is this problem for me only, or else is there any other way to do so,
> 
> Hoping with a positive response, **Regards, shiva**

https://sg.docworkspace.com/d/sIAfIncbkAbK-k7oG
[and-Conditions-for-DGA-Open-Government-LicenseEN04062021.pdf](https://github.com/user-attachments/files/17908602/and-Conditions-for-DGA-Open-Government-LicenseEN04062021.pdf)
